src/service/FileTransferService.ts

```diff
@@ -0,0 +1,46 @@
+import axios, { AxiosRequestConfig, AxiosResponse } from 'axios';
+import FormData from 'form-data';
+import { CREATED } from 'http-status-codes';
+
+export class FileTransferService {
+
+    private readonly uri: string;
+
+    constructor(private readonly host: string, private readonly key: string) {
+        this.host = host;
+        this.key = key;
+        this.uri = `${this.host}/dev/files`;
```

Comment:
Application will be configured with `FILE_TRANSFER_API_URL` which already contains path such as `dev/files`.